builds.json

```diff
@@ -113,6 +113,12 @@
           "python_package": "opencb",
           "version": "1.3.0-SNAPSHOT",
           "dependencies": []
+        },
+        {
+          "package": "org.opencb.biodata.models",
+          "python_package": "opencb",
+          "version": "1.2.0",
+          "dependencies": []
```

Comment:
@priesgo I can confirm this fixes the problems we saw with the tests failing on the latest CIP API tag